ezflow/functional/data_augmentation/augmentor.py

```diff
@@ -22,12 +22,14 @@ class FlowAugmentor:
     def __init__(
         self,
         crop_size,
+        crop_type="center",
```

Comment:
Is there a specific reason for using a string argument instead of a boolean?

Reply:
I just thought it conveys better that there are two types of cropping: center and random.